test/utils/describeConformance.js

```diff
@@ -427,7 +434,13 @@ function testThemeStyleOverrides(element, getOptions) {
         this.skip();
       }
 
-      const { muiName, classes, testStateOverrides, render } = getOptions();
+      const {
+        muiName,
+        classes,
+        testStateOverrides,
+        render,
+        ThemeProvider = MDThemeProvider,
```

Comment:
Did we settle on the naming conversions? How are we abbreviating Material Design "MUI" or "MD"?

Reply:
MD is better because the option we want to move forward on the docs "MUI Core as a product line — System, Joy, MD, and Base as products"